jest.config.js

```diff
@@ -1,7 +1,38 @@
 module.exports = {
-	roots: [`<rootDir>/assets/src`],
-	testMatch: [`**/?(*.)test.{ts,tsx}`],
+	roots: ['<rootDir>/assets/src'],
+	testMatch: ['<rootDir>/assets/src/**/*.test.{ts,tsx}'],
 	transform: {
-		'^.+\\.(js|jsx|ts|tsx)$': 'ts-jest',
+		'^.+\\.(js|jsx)$': '<rootDir>/node_modules/babel-jest',
+		'^.+\\.(ts|tsx)$': '<rootDir>/node_modules/ts-jest',
+		'^.+\\.css$': '<rootDir>/config/jest/cssTransform.js',
+		'^(?!.*\\.(js|jsx|ts|tsx|css|json)$)': '<rootDir>/config/jest/fileTransform.js',
 	},
+	collectCoverageFrom: ['assets/src/**/*.{js,jsx,ts,tsx}', '!src/**/*.d.ts'],
+	setupFiles: ['react-app-polyfill/jsdom'],
+	setupFilesAfterEnv: ['<rootDir>/config/jest/setupTests.ts'],
+	testEnvironment: 'jest-environment-jsdom-fourteen',
+	transformIgnorePatterns: ['[/\\\\]node_modules[/\\\\].+\\.(js|jsx|ts|tsx)$', '^.+\\.module\\.(css|sass|scss)$'],
+	modulePaths: [],
+	moduleNameMapper: {
```

Comment:
we might want to derive these aliases from tsconfig, the same way we did for webpack